broker-core/src/main/java/io/zeebe/broker/workflow/processor/instance/CreateWorkflowInstanceHandler.java

```diff
@@ -72,6 +72,10 @@ public void handle(WorkflowInstanceCommandContext commandContext) {
       if (handlerId != null && handlerId.capacity() != 0) {
         createDeferredStartEvent(
             handlerId, createWorkflowCommand, workflowInstanceKey, eventOutput);
+      } else if (workflowDefinition.getWorkflow().getStartEvents().size() > 1) {
```

Comment:
The command should be also rejected if the workflow has exactly one timer/message start event.